canonical-web-and-design/maas-ui
# refactor(preferences): migrate preferences to React Router v6

## Done

- Update the prefences routest to React Router v6.

## QA

### MAAS deployment

To run this branch you will need access to one of the following MAAS deployments:

- [Karura](/HACKING.md#karura)
- [Bolla](/HACKING.md#bolla)
- [A development MAAS](/HACKING.md#development-deployment)
- [A local snap MAAS](/HACKING.md#snap-deployment) (this will not usually have machines)

### Running the branch

You can run this branch by:

- Serving with [dotrun](/HACKING.md#maas-ui-development-setup)
- [Building in a development MAAS](/HACKING.md#running-maas-ui-from-a-development-maas)

### QA steps

- Click your username in the header.
- Check that you see the details form.
- Click on each item in the subnav and you should be able to see the lists as well as any add/edit forms.

## Fixes

Fixes: #4194.
